src/screens/AccountImportMethod/AccountImportMethod.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import React, { PureComponent } from 'react'
+import {
+  Image,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import textConstants from '../../locales/en'
+import PropTypes from 'prop-types'
+import TextButton from '../../components/TextButton'
+import styles from './AccountImportMethodStyles'
+
+export default class SelectAccountImportMethod extends PureComponent {
```

Comment:
Name of exported class must be the same as file name. Also I suggest to rename "AccountImportMethod" to "ImportMethod", because in this case our screens' names will be the same as on prototype (easy to search files)